docs/docs/configuration.md

```diff
@@ -10,15 +10,17 @@ description: >
 
 Setting configuration parameters at load-time is done by appending arguments after the `--loadmodule` argument when starting a server from the command line or after the `loadmodule` directive in a Redis config file. For example:
 
-In redis.conf:
+In [redis.conf](https://redis.io/docs/manual/config/):
```

Comment:
the link need to be relative like in other files

Reply:
Fixed